code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -94,6 +103,14 @@
 		to_chat(user,"<span class='notice'>The reagents inside \the [src] are already secured.</span>")
 	return
 
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/attackby(obj/item/weapon/W, mob/user)
+	if(W.isscrewdriver() && !is_open_container())
+		to_chat(user,"<span class='notice'>Using \the [W], you unsecure the autoinjector's lid.</span>") // it locks shut after being secured
+		flags |= OPENCONTAINER

```

Comment:
quick overview of recent changes doesn't show the reverse of this happening anywhere. can you actually close the hypospray?

Reply:
For one, it's an autoinjector, hyposprays themselves don't have this functionality. Yes, this is literally the opposite of the proc directly before it.